.changeset/sour-jobs-count.md

```diff
@@ -0,0 +1,8 @@
+---
+"@fluidframework/container-runtime": minor
+"@fluidframework/container-utils": minor
+---
+
+Remove DeltaManagerProxyBase
+
+`DeltaManagerProxyBase` was previously deprecated. It is no longer exported, and no replacement API is intended.
```

Comment:
Can you include the minor version this was deprecated in? It really helps partners. I know we're confident there are no usages of this, but we need to get into the habit of including these details to help partners trace changes.

Reply:
Yeah, that's a good call. Will update.

Reply:
JK. You already got it. Github's view is so inconsistent... Thank you!!